### DISCUSSION TITLE
Initializing Confusion

### DISCUSSION THREAD

**Discussion by Eronixy** [Znunu/EzMudae#5] [Q&A] [answered]
Hello there!

I have little experience in discord.py and I'm trying to use your code. From what I understand, I need to initialize the bot and mudae variable for the Mudae object to be created. I know that the bot = client but what do I input to the mudae variable? Whenever I use get_user it says User and str not supported. Please help me :<

**Comment by Znunu** [comment, marked as answer]
Hiya
Uhm, I'm not sure I exactly understand. You create the mudae object by passing it the mudae user. By get_user are you referring to discord.py client function? In that case, you should pass an int, not User or string. Int because user id is an integer. 

Can I add you on discord? It'd probably be quicker to solve your issue there.

Sorry for replying so late. I didn't realize you had asked this.

**Comment by Ian99559999898** [comment]
could you add me on discord? im confused as hell  
<img width="221" alt="image" src="https://github.com/Znunu/EzMudae/assets/165847121/f9e7290d-8c52-49e6-aefd-cde4356f64e0">


My username is " alwaysathome "
